web-www/src/sections/nct/LedgerSection.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { graphql, useStaticQuery } from 'gatsby';
+import SanityImage from 'gatsby-plugin-sanity-image';
+import BackgroundImage from 'gatsby-background-image';
+import { FluidObject } from 'gatsby-image';
+import { Box } from '@mui/material';
+
+import Section from 'web-components/lib/components/section';
+import Title from 'web-components/lib/components/title';
+import { BlockContent } from 'web-components/src/components/block-content';
+import { Label } from 'web-components/lib/components/label';
+import OutlinedButton from 'web-components/lib/components/buttons/OutlinedButton';
+
+import type { NctLedgerSectionQuery } from '../../generated/graphql';
+
+const query = graphql`
+  query nctLedgerSection {
+    bg: file(relativePath: { eq: "topo-bg-top.png" }) {
+      childImageSharp {
+        fluid(quality: 90) {
+          ...GatsbyImageSharpFluid_withWebp
+        }
+      }
+    }
```

Comment:
I thought about moving the topo background into sanity, but it seems like overkill. This could probably be simplified into a hook or css class (as we're doing in the registry app)